src/playermat/Playmat.ttslua

```diff
@@ -415,6 +415,10 @@ function replenishTokens(card, count, replenish)
     if obj.getCustomObject().image == "http://cloud-3.steamusercontent.com/ugc/1758068501357192910/11DDDC7EF621320962FDCF3AE3211D5EDC3D1573/" then
       foundTokens = foundTokens + math.abs(obj.getQuantity())
       obj.destruct()
+    elseif obj.getName() == "Resource Counter" then
+      foundTokens = obj.getVar("value")
+      obj.destruct()
```

Comment:
Why destroy the object here?  Should be able to just set the value on the current token, rather than destroying and creating a new one (which then has the value set on it)

Reply:
This was mostly done to minimize the code changes and have the same behaviour for "normal" and clickable resource tokens - replenish removes old tokens and then respawns the correct amount

Reply:
Updated